static/templates/recipient_row.handlebars

```diff
@@ -23,11 +23,11 @@
                 <a class="message_label_clickable narrows_by_topic"
                     href="{{topic_url}}"
                     title="{{#tr this}}Narrow to stream &quot;__display_recipient__&quot;, topic &quot;__topic__&quot;{{/tr}}">
-                    {{#if use_match_properties}}
-                    {{{match_topic}}}
-                    {{else}}
-                    {{topic}}
-                    {{/if}}
+                    {{~#if use_match_properties~}}
+                    {{~ match_topic ~}}
```

Comment:
Not sure if these triple braces we're intentionally untripled. (Triple braces mean "don't escape HTML".)

Reply:
It's double braces with a tilde, correct?  The tilde does whitespace removal, I believe.

Reply:
@showell The tilde is not in question, I was the one who [suggested](https://chat.zulip.org/#narrow/stream/9-issues/subject/double.20topic.3F/near/621828) that in the first place.

This was previously `{{{match_topic}}}`, and the question is whether the previous triple braces were a mistake (hence @synicalsyntax intentionally untripled them to `{{~ match_topic ~}}`), or whether the existing triple braces were intentional (hence this should have been `{{{~ match_topic ~}}}`).